vowpalwabbit/core/src/reductions/cb/cb_explore_adf_large_action_space.cc

```diff
@@ -314,6 +315,10 @@ VW::LEARNER::base_learner* VW::reductions::cb_explore_adf_large_action_space_set
       .add(make_option("model_weight", model_weight_impl))
       .add(make_option("vanilla", use_vanilla_impl))
       .add(make_option("full_spanner", full_spanner))
+      .add(make_option("thread_pool_size", thread_pool_size)
+               .default_value(0)
```

Comment:
if we want multithreading by default, perhaps use `std::thread::hardware_concurrency`